examples/multioutput/plot_classifier_chain_yeast.py

```diff
@@ -92,19 +92,27 @@
                'Chain 10',
                'Ensemble Average')
 
-y_pos = np.arange(len(model_names))
-y_pos[1:] += 1
-y_pos[-1] += 1
+x_pos = np.arange(len(model_names))
+x_pos[1:] += 1
+x_pos[-1] += 1
 
 # Plot the Jaccard similarity scores for the independent model, each of the
 # chains, and the ensemble (note that the vertical axis on this plot does
 # not begin at 0).
 
-fig = plt.figure(figsize=(7, 4))
-plt.title('Classifier Chain Ensemble')
-plt.xticks(y_pos, model_names, rotation='vertical')
-plt.ylabel('Jaccard Similarity Score')
-plt.ylim([min(model_scores) * .9, max(model_scores) * 1.1])
+fig, ax = plt.subplots()
```

Comment:
I would keep the syntax `fig = plt.figure(figsize=(7, 4))`

You seem to prefer `fig = plt.figure(figsize=(7, 7))`,
I would rather go with something more compact, but it's a matter of taste

To have more compact xlabels, you could also add `\n` in the long model names:
`'Independent Models'` -> `'Independent\nModels'`